scipy/stats/tests/test_discrete_basic.py

```diff
@@ -118,20 +119,20 @@ def test_rvs_broadcast(dist, shape_args):
     shape_only = dist in ['betabinom', 'betanbinom', 'skellam', 'yulesimon',
                           'dlaplace', 'nchypergeom_fisher',
                           'nchypergeom_wallenius']
+    if dist in {'poisson_binom'}:
+        pytest.skip("Distribution doesn't follow typical broadcasting rules")
 
     try:
         distfunc = getattr(stats, dist)
     except TypeError:
         distfunc = dist
         dist = f'rv_discrete(values=({dist.xk!r}, {dist.pk!r}))'
     loc = np.zeros(2)
-    nargs = distfunc.numargs
```

Comment:
These changes can also be undone, although they are arguably more Pythonic. Will push later.